src/relayserver/runServer.js

```diff
@@ -13,6 +13,23 @@ function error (err) {
   process.exit(1)
 }
 
+/// TODO: remove once translated to TypeScript. This method is in ./cli/utils.ts
+function getRelayHubAddress (hub) {
```

Comment:
you ignore the function's logic if a parameter is given?
why not `const hub = commander.hub ?? getRelayHubAddress()`

(who approved that cli.ts ?)

Reply:
Yes, you're right. Will fix. On next PR.